packages/ast/src/encoding/proto/types.ts

```diff
@@ -135,6 +135,7 @@ export const getEnumFromJsonName = (name) => {
 };
 
 export const getFieldsTypeName = (field: ProtoField) => {
+    if (!field?.scope) return field.parsedType.name;
```

Comment:
this may be an issue, we may want to pass in scopes manually from the top when we loop over fields somewhere